example/lib/main.dart

```diff
@@ -66,14 +65,32 @@ class _MyAppState extends State<MyApp> {
         _variableValue = updatedValue;
       });
     });
+
+    final booleanVariable =
+        await _dvcClient.variable('boolean-variable', false);
+    setState(() {
+      _booleanValue = booleanVariable?.value;
+    });
+    booleanVariable?.onUpdate((updatedValue) {
+      setState(() {
+        _booleanValue = updatedValue;
+      });
+    });
   }
 
   void resetUser() {
     _dvcClient.resetUser();
+    setState(() {
+      _displayValue = 'User reset!';
+    });
   }
 
   void identifyUser() {
```

Comment:
can we also add a button + method to identifyUser with an `isAnonymous(true)` or no `userId` provided?